pandas/core/groupby/groupby.py

```diff
@@ -518,7 +518,8 @@ class providing the base-class of operations.
     - function
     - string function name
     - list of functions and/or function names, e.g. ``[np.sum, 'mean']``
-    - dict of axis labels -> functions, function names or list of such.
+    - dict of axis labels -> functions, function names or list of such,
+      e.g. ``{{'col1': 'min', 'col2': list}}``.
```

Comment:
1. Could you add this to the `Example` section instead of here?
2. We tend to discourage container data structures, e.g. list, in DataFrames. Maybe better to demonstrate a another callable function like `lambda x: sum(x) + 2`

Reply:
Sure!